resolwe/flow/models/entity.py

```diff
@@ -26,3 +27,39 @@ class Meta(BaseCollection.Meta):
 
     #: indicate whether `descriptor` is completed (set by user)
     descriptor_completed = models.BooleanField(default=False)
+
+
+class RelationType(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+
+    ordered = models.BooleanField(default=False)
```

Comment:
> Idea is that client (or package with defined relation types) takes care of ordering.

But in this case, the backend cannot return entities in proper order and all sorting must be done on the client? What if there are a lot of entities and the client only wants a subset? Or is this not a supported use case?